### DISCUSSION TITLE
Chias do not leave the Chia Wallet "Please enter a valid numeric amount"

### DISCUSSION THREAD

**Discussion by marcocastellobranco** [Chia-Network/chia-blockchain#5783] [Support]
I am unable to send my Wallet Chias to gate.io, when I enter the withdrawal amount, the message is "Please enter a valid numeric amount" moreover, even with the synchronized wallet, and all the amount I have in chia on walltet having the transaction confirmed, the amount still appears in the "Pending Total Balance"
How do I overcome this "valid numeric amount" problem to send to the gate? Thank you guys

**Comment by BrandtH22** [comment]
@marcocastellobranco , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
